freqtrade/exchange/exchange.py

```diff
@@ -624,8 +624,8 @@ def get_order(self, order_id: str, pair: str) -> Dict:
         try:
             return self._api.fetch_order(order_id, pair)
         except ccxt.InvalidOrder as e:
-            raise DependencyException(
-                f'Could not get order. Message: {e}')
+            raise InvalidOrderException(
+                f'Tried to get an invalid order. Message: {e}')
```

Comment:
More a question than anything - should we log the order-id here? 

I guess it would simplify certain error-debuggings if we know what order it tried to get ...

Reply:
your point is valid.